README.md

```diff
@@ -47,6 +47,9 @@ Currently, supported relations are:
   - [Prometheus](https://github.com/canonical/prometheus-operator), which forwards alerts to
     Alertmanager over the `alertmanager_dispatch` interface. Set up with: 
     `juju add-relation alertmanager-k8s prometheus-k8s`.
+  - [Karma](https://github.com/canonical/karma-operator), which displays alerts from all related alertmanager instances 
+    in a nice dashboard.
```

Comment:
```
in [Karma](https://github.com/prymitive/karma), which is charmed by the [Karma operator](https://charmhub.io/karma-k8s).
```

Reply:
docstring improved to match the prometheus line above.